ios/browserstack_sample_local.py

```diff
@@ -6,63 +6,15 @@
 from browserstack.local import Local
```

Comment:
this should not be there - since SDK will handle local

Reply:
removed